server/src/DBTablesMonitoring.cc

```diff
@@ -1368,6 +1375,11 @@ const std::set<EventType> &EventsQueryOption::getEventTypes(void) const
 	return m_impl->eventTypes;
 }
 
+void EventsQueryOption::setEventIds(const std::list<EventIdType> &eventIds)
```

Comment:
Could you remove redundant `std::` namespace prefix?
Hatohol's server side C++ source normally use `using namespace std;` on top level.